pkg/config/remote/client/client.go

```diff
@@ -436,6 +436,10 @@ func (c *Client) pollLoop() {
 					log.Errorf("could not update remote-config state: %v", c.lastUpdateError)
 				}
 			} else {
+				if !successfulFirstRun {
+					log.Infof("first update successful")
```

Comment:
Is this variable misnamed? "if not successful first run, log successful first update" ?

Reply:
I forgot to remove the `!`

Reply:
We unconditionally on line 442 debug log that it was successful right after this - so do we really need this check?